.github/semantic.yaml

```diff
@@ -0,0 +1,44 @@
+---
+# Always validate the PR title, and ignore the commits
+# titleOnly: true
+
+# Always validate all commits, and ignore the PR title
+commitsOnly: true
+
+# Always validate the PR title AND all the commits
+# titleAndCommits: true
+
+# Require at least one commit to be valid
+# this is only relevant when using commitsOnly: true or titleAndCommits: true,
+# which validate all commits by default
+# anyCommit: true
+
+# Allow use of Merge commits (eg on github: "Merge branch 'master' into
+# feature/ride-unicorns")
+# this is only relevant when using commitsOnly: true (or titleAndCommits: true)
+allowMergeCommits: false
+
+##
+## We do not use scopes, list is empty.
+##
+# You can define a list of valid scopes
+# scopes:
+#  - scope1
+#  - scope2
+
+# By default types specified in commitizen/conventional-commit-types is used.
+##
+## We override the types by components we have.
+##
+types:
```

Comment:
need to update the development guide for this one.

Reply:
yes, but I prefer to get this in before, so that all contributors have some standard to follow.